pom.xml

```diff
@@ -107,78 +101,69 @@
 			<artifactId>spring-boot-starter-actuator</artifactId>
 		</dependency>
 		<dependency>
-		  <groupId>io.fabric8</groupId>
-		  <artifactId>kubernetes-client</artifactId>
-		  <version>5.3.1</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>kubernetes-client</artifactId>
+			<version>5.3.2</version>
 		</dependency>
 		<dependency>
-		    <groupId>io.fabric8</groupId>
-		    <artifactId>tekton-client</artifactId>
-		    <version>5.5.0</version>
+			<groupId>io.fabric8</groupId>
+			<artifactId>tekton-client</artifactId>
+			<version>5.5.0</version>
```

Comment:
We could try moving to 5.12.2 of the fabric8 client, or I can do on a separate branch and PR it in?

this would apply to most of the fabric8 artifacts I think